test/lib/rule-finder.js

```diff
@@ -2,13 +2,14 @@ const path = require('path');
 const createRequire = require('create-require');
 const assert = require('assert');
 const proxyquire = require('proxyquire');
-
+const { builtinRules } = require('eslint/use-at-your-own-risk');
 const semver = require('semver');
-const eslintPkg = require('eslint/package.json');
+const merge = require('lodash.merge');
+const { ESLint } = require('eslint');
 
 const processCwd = process.cwd;
-
-const eslintVersion = semver.satisfies(eslintPkg.version, '< 5') ? 'prior-v5' : 'post-v5';
```

Comment:
seems like this should be moving the post-v5 fixtures dir contents to `prior-v8`?

Reply:
Yes.
I'm moving the post-v5 data directly to prior-v8. This allows us to test both config definitions up to v8 and v9 config definitions.